.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,33 @@
+name: Tests
+
+on:
+  pull_request:
+  push:
+    branches: [master, main]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php: [7.4, 8.0, 8.1]
+
+    name: PHP ${{ matrix.php }} - ${{ matrix.stability }} Test
```

Comment:
What is `{{matrix.stability}}`?

Reply:
That would be an old parameter from the matrix I forgot to remove!

Reply:
That would be an old parameter from the matrix I forgot to remove!